main.py

```diff
@@ -54,6 +56,15 @@ def get_user(self, id: int) -> BotUser:
         return self.users[id]
 
 
+class Device:
+    def __init__(self, device: dict):
+        self.id: int = int(device["id"])
+        self.label: str = device['label']
+        self.type: str = device['type']
+        self.commands: list[str] = device['commands'] or []
```

Comment:
I am not sure what's the intention of `device['commands'] or []`, but if there is no `'commands'`, the dictionary will raise KeyNotFound exception, not return None.  If one wants the other behavior, use `device.get('commands', [])`

Reply:
looks like it's not needed. removing